pkgs/development/libraries/ncurses/default.nix

```diff
@@ -46,6 +46,9 @@ stdenv.mkDerivation rec {
   buildInputs = lib.optional (mouseSupport && stdenv.isLinux) gpm;
 
   preConfigure = ''
+    # These paths end up in the default lookup chain.
+    export TERMINFO_DIRS=/etc/terminfo
+
     export PKG_CONFIG_LIBDIR="$dev/lib/pkgconfig"
```

Comment:
Quoting _is_ appropriate here because if expanded `$dev` needs quoting, then it will break the script. Quoting can be safely omitted when using `export` like this: `a=$b; export a`.